### ISSUE TITLE
Reveal presentation: strange behavior using columns and heading 1

### ISSUE THREAD

**Issue by aborruso** [quarto-dev/quarto-cli#824]
Hi all,
I'm creating a presentation both with horizontal and vertical slides.

When I insert a slide using heading 1, with inside some columns, I have a wrong rendering of the title.

![image](https://user-images.githubusercontent.com/30607/167251164-27d9ca19-4656-4098-aad4-a5e957e526bd.png)

A slide with columns works only if I use a heading 2 slide after a heading 1 one.

You can see it in action here <https://bl.ocks.org/aborruso/raw/e3078894862fa45a7dd16fb919f50282/?raw=true#/title-slide>.

This is the code of the presentation: <https://gist.github.com/aborruso/aa0e0a0455e5e9e88420dba745ab6bbe>.

Thank you

**Comment by jjallaire** [comment]
Yes, currently columns don't play well with title slides (as the title slide wants to center its heading horizontally).

**Comment by jjallaire** [comment]
A workaround:

1) Specify the option `center-title-slide: false`

2) Modify the `columns` div as follows: `::: {.columns style="margin-top: 90px;"}`

**Comment by aborruso** [comment]
Thank yoi @jjallaire 

> A workaround:
> 
> 1. Specify the option `center-title-slide: false`
> 2. Modify the `columns` div as follows: `::: {.columns style="margin-top: 90px;"}`

Not so good

![image](https://user-images.githubusercontent.com/30607/167262662-ead9a953-360a-4baf-9d30-be09a33dcf79.png)

**Comment by aborruso** [comment]
@jjallaire my error, it works.

But for me it is not so obvious, that I must use

```
::: {.columns style="margin-top: 120px;"}
```

and that this does not work

```
::: columns {style="margin-top: 90px;"}
```

**Comment by jjallaire** [comment]
Okay, glad you got it to work!

**Comment by aborruso** [comment]
I have solved, using HTML for heading and not the `#`.

Using `<h3>Colum name</h3>` and not `### Column name`
